### ISSUE TITLE
Spec: "it should transform given answers" looks like an ineffective test

### ISSUE THREAD

**Issue by warang580** [FreeAllMedia/stimpak#34]
I think the spec is wrong because `transformed` is already set to `5` (and not `"5"`) when assignated, and not modified through `transform()`.

``` javascript
        let answers = {
            transformed: 5,
            untouched: "5"
        };

        stimpak
            .answers(answers)
            .transform(callbackA)
            .generate(error => {
                stimpak.answers().transformed.should.eql(5);
                stimpak.answers().untouched.should.eql("5");

                done(error);
            });
```

I think this is how it should be written : https://github.com/FreeAllMedia/stimpak/compare/develop...warang580:fix/transform

But this spec doesn't pass. It looks like you are only using `transforms()` if there are prompts to show. I didn't found a working way to change it for now.

**Comment by warang580** [comment]
Also, the Stimpak `transform()` documentation is not correct in the [API](https://github.com/FreeAllMedia/stimpak/blob/develop/API.md#transformtransformingfunction).
It looks like it's a copy/paste that you forgot to edit.

**Comment by dcrockwell** [comment]
FYI, this is only a renamed `.cast` feature. Renamed because it does more than just casting.

**Comment by dcrockwell** [comment]
Closed by 2e03bba14704143f42840285b1e25d2bd1b54c46

**Comment by dcrockwell** [comment]
Fixed version released: https://github.com/FreeAllMedia/stimpak/releases/tag/0.1.1
